common/changes/@rushstack/heft/user-danade-JestPlugin_2021-05-28-19-52.json

```diff
@@ -0,0 +1,11 @@
+{
+  "changes": [
+    {
+      "packageName": "@rushstack/heft",
+      "comment": "Remove Jest plugin from Heft. To consume the Jest plugin, add @rushstack/heft-jest-plugin as a dependency and include it in heft.json",
+      "type": "major"
```

Comment:
Heft 1.0 hasn't been released yet, so for now this should be a MINOR bump. (We probably should make a plan to go to 1.0 though. And Rush 6)

Reply:
```suggestion
      "type": "minor"
```